addons/addons.json

```diff
@@ -162,6 +161,7 @@
   "material-forum-editor-buttons",
   "forum-quote-code-beautifier",
   "horizontal-mystuff-tabs",
+  "colorblind",
```

Comment:
Does moving this addon down here have any negative consequences?

Reply:
I think it was for something on a page with `scratchr2` enabled but moving it down also broke some other styles, so I switched to just using `!important`.